Docs/source/usage/parameters.rst

```diff
@@ -2985,6 +2985,56 @@ Reduced Diagnostics
         1 or 0, it is possible to compute the charge on only some part of the
         embedded boundary.
 
+    * ``Collider-relevant``
```

Comment:
The existing options seem to use CamelCase, so maybe for consistency:
```suggestion
    * ``ColliderRelevant``
```
(+ impl below :) )